packages/alert-banner/src/AlertBanner.ts

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import {
+    CSSResultArray,
+    html,
+    SpectrumElement,
+    TemplateResult,
+} from '@spectrum-web-components/base';
+import { property } from '@spectrum-web-components/base/src/decorators.js';
+import '@spectrum-web-components/button/sp-close-button.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-alert.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-info.js';
+import styles from './alert-banner.css.js';
+
+export type AlertBannerVariants = 'neutral' | 'info' | 'negative' | '';
+const VALID_VARIANTS = ['neutral', 'info', 'negative'];
```

Comment:
```suggestion
const VALID_VARIANTS = ['neutral', 'info', 'negative'];
export type AlertBannerVariants = typeof VALID_VARIANTS[number] | '';
```
Less text duplication.

Reply:
Also is `''` technically valid? If so, should it be included in the `VALID_VARIANTS` array?

Reply:
It's valid in the sense that the `spectrum-css` styling will default to neutral but I wouldn't set `''` as valid variant, we should probably remove the attribute in this case

Reply:
In that case is it actually one of the `AlertBannerVariants`?

Reply:
Nope, I will remove it from `AlertBannerVariants`